ports/samd/machine_pin.c

```diff
@@ -54,6 +54,23 @@ typedef struct _machine_pin_irq_obj_t {
     uint8_t pin_id;
 } machine_pin_irq_obj_t;
 
+#if MICROPY_HW_PIN_BOARD_CPU
```

Comment:
I think this option would be better named `MICROPY_PY_MACHINE_PIN_BOARD_CPU`.

Reply:
Changed.